ENIGMAsystem/SHELL/Makefile

```diff
@@ -79,7 +79,10 @@ endif
 
 # CPPFLAGS needs these include dirs unconditionally
 override CPPFLAGS += $(SYSTEMS:%=-I%/Info)
-override CPPFLAGS += -I. -I$(CODEGEN)
+override CPPFLAGS += -I. -I$(CODEGEN) -I../../lodepng
```

Comment:
Please put lodepng in a directory called "shared" instead of just shoving it in the root of  the repository. Then extract "../../shared/" into a SHARED_SOURCES variable, and use `-I$(SHARED_SOURCES)/lodepng`.

Reply:
Taken care of: 4c64f7fa6b1c0e80cac8fc25f67287cd459bf42d